docs/cloud/bitbucket-server-oauth1-integration-setup.md

```diff
@@ -1,6 +1,16 @@
 # Bitbucket Data Center / Server Integration Setup (OAuth 1)
 ## Private Packagist Cloud
 
+<div class="row column">
+    <div class="callout warning">
+        <p>
+            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.
+            Consult the <a href="/docs/cloud/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.
+            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```

Comment:
```suggestion
            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.<br>
            Consult the <a href="/docs/cloud/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.<br>
            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```
I guess you wanted that?

Reply:
Not really. I just added new lines to make it easier work with. If you think that the line breaks help with reading it in a web page then let's add them

Reply:
Added the second line break but removed the first